apps/nextra/pages/en/build/create-aptos-dapp/faq.mdx

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
You need to create a `_meta.tsx` at the root of the `create-aptos-dapp` folder with `templates`. Right now it's lowercase in the sidebar